scripts/pegasus_rebalance_cluster.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+#
+# Pegasus cluster rebalance 
+#
+
+PID=$$
+
+if [ $# -le 2 ]; then
+  echo "USAGE: $0 <cluster-name> <cluster-meta-list> <only_move_primary>"
+  echo
+  echo "for example:"
+  echo "  $0 onebox 127.0.0.1:34601,127.0.0.1:34602 true"
+  echo
+  exit 1
+fi
+
+cluster=$1
+meta_list=$2
+only_move_primary=$3
+
+pwd="$( cd "$( dirname "$0"  )" && pwd )"
+shell_dir="$( cd $pwd/.. && pwd )"
+cd $shell_dir
+
+source ./scripts/minos_common.sh
+find_cluster $cluster
+if [ $? -ne 0 ]; then
+  echo "ERROR: cluster \"$cluster\" not found"
+  exit 1
+fi
+
+echo "UID=$UID"
+echo "PID=$PID"
+echo "Start time: `date`"
+rebalance_start_time=$((`date +%s`))
+echo
+
+echo "Generating /tmp/$UID.$PID.pegasus.rebalance.cluster_info..."
+echo cluster_info | ./run.sh shell --cluster $meta_list 2>&1 | sed 's/ *$//' >/tmp/$UID.$PID.pegasus.rebalance.cluster_info
+cname=`grep zookeeper_root /tmp/$UID.$PID.pegasus.rebalance.cluster_info | grep -o '/[^/]*$' | grep -o '[^/]*$'`
```

Comment:
It's very hard to read these code, JSON format output is supported, could you plz to use JSON?

Reply:
We are also plan to refactor remote commands to HTTP APIs, the latter will output in JSON format, we can reuse these JSON parser code then.

Reply:
Well, now all scripts use the `regex` to filter the result, I think they can be refactored uniformly later

Reply:
OK, could you please add an issue to trace this plan?

Reply:
Yeah, I have add it https://github.com/apache/incubator-pegasus/issues/578